src/main/java/frc/robot/Robot.java

```diff
@@ -120,6 +262,8 @@ public void teleopInit() {
     new DriveCommand().start();
```

Comment:
Are you planning to drive through Sandstorm?  If so, these will need to be run in autonomousInit().     teleopInit() will not start until the end of sandstorm

Also, this file seems to be missing the autonomousPeriodic and teleopPeriodic, both of which need to run the Scheduler

Reply:
The file does include the [auto methods](https://github.com/poly-rabbotics/2019-Deep-Space/blob/f21965d3cfbda66b14802de1d0dba7259b6277d9/src/main/java/frc/robot/Robot.java), GitHub is only showing a diff not the whole file. `autonomusInit()` does need to have a `DriveCommand` though